recipes/diligent-tools/all/conanfile.py

```diff
@@ -48,15 +63,17 @@ def configure(self):
             del self.options.fPIC
 
     def _patch_sources(self):
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            tools.patch(**patch)
+        patches.apply_conandata_patches(self)
 
     def validate(self):
-        if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, 11)
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, 11)
         if self.options.shared:
             raise ConanInvalidConfiguration("Can't build diligent tools as shared lib")
 
+    def build_requirements(self):
+        self.tool_requires("cmake/3.24.2")
+
```

Comment:
```suggestion
```

Not needed: https://github.com/DiligentGraphics/DiligentTools/blob/API252005/CMakeLists.txt#L1

Reply:
build 9 failed before I added this line.
Now it is working